### PR TITLE
More widget tests

### PR DESCRIPTION
**Pull request by DonJayamanne**

### CODE REVIEW

File: src/test/datascience/widgets/standard.vscode.test.ts [hunk 1]
```diff
@@ -73,32 +76,111 @@ suite('Standard IPyWidget (Execution) (slow) (WIDGET_TEST)', function () {
...
+                const innerHTML = await comms.queryHtml('.widget-checkbox', cell.outputs[0].id);
+                assert.include(innerHTML, 'Check me');
+                assert.include(innerHTML, '<input type="checkbox');
+                return true;
+            },
+            WidgetRenderingTimeoutForTests,
+            'Checkbox not rendered'
+        );
+    });
+    test.skip('Widget renders after executing a notebook which was saved after previous execution', async () => {
```
Comment (DonJayamanne) [inline src/test/datascience/widgets/standard.vscode.test.ts:134]: Doesn't work, will need to fix this (possibly something wrong in our code)
Looks like a regression.
Works only if you wait for some time

Comment (IanMatthewHuff) [reply src/test/datascience/widgets/standard.vscode.test.ts:134]: Does the test need to be skipped and an issue filed then? Or were you looking at this fix as part of this PR?

Comment (IanMatthewHuff) [reply src/test/datascience/widgets/standard.vscode.test.ts:134]: Whoops, totally missed the skip that was already there.

**Review by IanMatthewHuff — APPROVED** [review]

File: src/test/datascience/widgets/standard.vscode.test.ts (hunk 1 above)
Comment (rchiodo) [reply src/test/datascience/widgets/standard.vscode.test.ts:134]: Seems like a regression? Didn't we have this bug before?
https://github.com/microsoft/vscode-jupyter/issues/8431

**Review by rchiodo — APPROVED** [review]

File: src/test/datascience/widgets/standard.vscode.test.ts (hunk 1 above)
Comment (DonJayamanne) [reply src/test/datascience/widgets/standard.vscode.test.ts:134]: Aah, yes its a duplicate then.
This wasn't fixed, its actually flaky, it works locally (seems to be 100% of the time).
Must be some race condition (i think thats obvious).

Will re-open #8431
File https://github.com/microsoft/vscode-jupyter/issues/8749 & https://github.com/microsoft/vscode-jupyter/issues/8749

I.e. we have 3 issues related to widgets, but they all seem to work locally (at least for me)
Possible its just flaky

Comment (DonJayamanne) [reply src/test/datascience/widgets/standard.vscode.test.ts:134]: Damn it, looks like restart tests are also flaky

**Review by amunger — APPROVED** [review]

**Pull request merged**